code/modules/mob/living/blood.dm

```diff
@@ -10,7 +10,7 @@
 	if(NOBLOOD in dna.species.species_traits || HAS_TRAIT(src, TRAIT_NOBLEED) || (HAS_TRAIT(src, TRAIT_FAKEDEATH)))
 		return
 
-	if(bodytemperature < TCRYO || (HAS_TRAIT(src, TRAIT_HUSK))) //cryosleep or husked people do not pump the blood.
+	if(bodytemperature < 225 || (HAS_TRAIT(src, TRAIT_HUSK))) //cold or husked people do not pump the blood.
```

Comment:
I don't understand why you removed the cryo define when you're still using the number

Reply:
(AKA fix your magic numbers)

Reply:
Tcryo is misleading and the number is only used once. Ill make it a define though one second